csharp/Facebook.Yoga/Facebook.Yoga.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net6.0-ios</TargetFrameworks>
+    <IsPackable>true</IsPackable>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="NuGetizer" Version="0.9.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <SupportedRid Include="win-x64;linux-x64" />
+    <PackageFile Include="@(SupportedRid->'Facebook.Yoga.Native.%(Identity)')" PackFolder="Dependency" Version="$(Version)" />
```

Comment:
```suggestion
    <PackageFile Include="@(SupportedRid->'Facebook.Yoga.Native.%(Identity)')" PackFolder="Dependency" Version="$(Version)" PackageLicenseExpression="MIT" />
```

Reply:
I don't think you specify the license expression on the dependency. I'll add it to the project files for each package.